package.json

```diff
@@ -19,7 +19,7 @@
     "json-stringify-pretty-compact": "^4.0.0",
     "probe-image-size": "^7.2.3",
     "sanitize-html": "^2.17.0",
-    "smogon": "^3.1.20",
```

Comment:
please revert the changes to the version numbers, when i cut a release i bump them